src/control/drpc/modules_test.go

```diff
@@ -9,22 +9,12 @@ package drpc
 import (
 	"testing"
 
-	"github.com/golang/protobuf/proto"
 	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/proto"
 
 	"github.com/daos-stack/daos/src/control/common"
 )
 
-func TestMarshal_Failed(t *testing.T) {
```

Comment:
why was this removed?

Reply:
Doesn't fail anymore. Calling `proto.Marshal(nil)` just returns nil with no error.